### ISSUE TITLE
[49.17] Missing SPAs from MekHQ

### ISSUE THREAD

**Issue by RaozSpaz** [MegaMek/megamek#5179]
### Environment
MekHQ 49.17 [49.18 just released and I have not checked it yet]
Windows
Java 11

### Description
There are a series of SPAs in the Campaign Ops book that are not available in the Mek HQ / MegaMek. There are some on this list that most likely cant be properly included in general but I will still place them in the list just so its comprehensive. I do not own the C.Ops book but was provided snips of the tables with the SPAs on them for use in a campaign setting.  If I get the provider to give me the page number I will add it ASAP. None the less, the list:

BASIC SPAS

- [ ] Antagonizer
- [ ] Combat Intuition
- [ ] Demoralizer
- [ ] Fist Fire
- [ ] Heavy Lifter
- [ ] Marksman
- [ ] Natural Grace
- [ ] Range master
- [ ] Sharpshooter
- [ ] Slugger
- [ ] Speed Demon
- [ ] Stand Aside
- [ ] Street Fighter
- [ ] Swordsman
- [ ] Terrain Master [Drag Racer]

AIRBORNE SPAS

- [ ] Dust Off
- [ ] Ground Hugger
- [ ] Ride The Wash
- [ ] Wind Walker

INFANTRY SPAS

- [ ] Heavy Horse
- [ ] Light Horseman

Beast Infantry were just added so it makes sense why those two SPAs aren't in the game. Antagonizer and Demoralizer I doubt could be utilized properly in MM. But a few of the others should/could work. Either way my job was just to upload the list. Hopefully it is helpful.

**Comment by HammerGS** [comment]
Thanks for compiling the list.

**Comment by SuperStucco** [comment]
Range Master should be available, it's in at least as far back as 49.12.

**Comment by RaozSpaz** [comment]
I will double check in a moment. I didnt see it at a glance but i mightve missed it

**Comment by RaozSpaz** [comment]
They are correct. I missed Range Master and it is already in. My apologies.

**Comment by RaozSpaz** [comment]
I would assume Wind Walker should be relatively easy. At least in its short description it mainly disables atmospheric penalties. The landing and takeoff penalties may be harder not certain. But even just the first half of the SPA would be a pretty big addition.

**Comment by pheonixstorm** [comment]
Wind Walker would only affect landings unless the space/atmosphere interface is a thing now. Shouldn't be too difficult even for a beginner.

**Comment by RaozSpaz** [comment]
Does it not remove the penalties to your Control Rolls for operating in Atmo?

**Comment by pheonixstorm** [comment]
No, it only states a -1 to landing rolls including crash landing and when passing the space/atmo interface. The text hasn't changed since the first printing.

**Comment by celestiun** [comment]
bumping this post. As many of these are not implemented still in 49.19.1
Speed demon specifically as it is like 50% of the formation SPA bonuses from Campaign OPs. and just increases movement a bit.

**Comment by ncrt10** [comment]
Fist Fire is such a fun SPA to design mechs around, that it's unfortunate that it's not implemented yet.
